### ISSUE TITLE
`WebApp` Example

### ISSUE THREAD

**Issue by Bibo-Joshi** [python-telegram-bot/python-telegram-bot#3021]
It would be nice to have an example for the `WebApps` added in Bot API 6.0. This example should

* showcase both inline & reply keyboard buttons for starting the web app
* showcase how to set the menu button to open the web app
* include a minimal website that can be used for the example, i.e. a HTML file with the necessary JS included. That file can probably be hosted on python-telegram-bot.org for demonstration purposes
* be based on `master`, i.e. for v20.x. We can consider a backport to v13 once the example is merged & api 6.0 is backported to v13

What the web app actually does is not overly important, but it would be good if it would accept some kind of user interaction. One idea would to show a color picker and let the user select a color, e.g. with [this library](https://iro.js.org). That's an interface that's hard to replicate in the native TG GUI and with iro.js it should be little work to integrate into a webapp.

**Comment by Bibo-Joshi** [comment]
closed by #3052
